conda/activate.py

```diff
@@ -708,6 +708,18 @@ def _build_activate_shell_custom(self, export_vars):
     def _hook_preamble(self):
         raise NotImplementedError()
 
+    def _add_prefix_to_path(self, prefix, starting_path_dirs=None):
+        # If this is the first time we're activating an environment, we need to ensure that
```

Comment:
Is it really only the first time?
If you'd do
```
conda activate base
conda activate some-other-env
conda activate base
conda activate some-other-env
```
wouldn't `conda.exe` precede `condacmd/conda.bat` on `PATH` after the second `conda activate base` so that the last `conda activate some-other-env` fails?
I'm not really sure on this though and can't check that right now. Maybe create a test case for that?